src/commands/reminder/command.ts

```diff
@@ -0,0 +1,79 @@
+import { addMilliseconds, formatDistanceToNow, isBefore } from "date-fns";
```

Comment:
as a random note, interacting with DB here should probably be abstracted out to an API. However this is fine for now

Reply:
data access methods can probably be added along side the entity, but i agree for now this is fine.